.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
+cache: pip
+install:
+  - pip install -r requirements.txt
+script:
+  - python test.py
```

Comment:
I see.

But then with the current setup, I am not sure of the advantages of having a CI.

One solution would be to implement a `--watch/-w` flag in `build.py`, that will only watch the files when its specified